example-get-started-cv/code/notebooks/TrainSegModel.ipynb

```diff
@@ -0,0 +1,270 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_arch = 'resnet18'\n",
+    "\n",
+    "for base_lr in [0.001, 0.005, 0.01]:\n",
+    "    live = Live(dir=os.path.join('results', 'train'), \n",
+    "            report=\"md\", \n",
+    "            save_dvc_exp=True)\n",
+    "    live.log_param(\"base_lr\", base_lr)\n",
+    "    learn = unet_learner(data_loader, \n",
+    "                        arch=getattr(models, train_arch), \n",
+    "                        metrics=DiceMulti)\n",
+    "    fine_tune_args = {\n",
+    "        'epochs': 8,\n",
+    "        'base_lr': base_lr\n",
+    "        }\n",
+    "    learn.fine_tune(\n",
+    "        **fine_tune_args,\n",
+    "        cbs=[DVCLiveCallback(live=live)])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Compare experiments\n",
+    "!dvc exp show --only-changed"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Apply best performing experiment to the workspace\n",
+    "!EXP=$(dvc exp show --csv --sort-by dice_multi | tail -n 1 | cut -d , -f 1) && dvc exp apply $EXP"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Review sample preditions vs ground truth"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "learn.show_results(max_n=6, alpha=0.7)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Review instances where loss function values are the highest (i.e. model is likely to be wrong)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from fastai.vision.all import SegmentationInterpretation\n",
+    "\n",
+    "interp = SegmentationInterpretation.from_learner(learn)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "interp.plot_top_losses(k=5, alpha=0.7)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.10.4 (main, Jan 25 2023, 00:13:50) [GCC 9.4.0]"
+  },
+  "orig_nbformat": 4,
+  "vscode": {
+   "interpreter": {
+    "hash": "3ad933181bd8a04b432d3370b9dc3b0662ad032c4dfaa4e4f1596c548f763858"
+   }
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```

Comment:
TODO @alex000kim Can we also create `results/evaluate` in the notebook?

Reply:
https://github.com/iterative/example-repos-dev/pull/162